packages/mui-material/src/Rating/Rating.d.ts

```diff
@@ -1,17 +1,17 @@
 import * as React from 'react';
 import { SxProps } from '@mui/system';
 import { OverridableStringUnion } from '@mui/types';
-import { InternalStandardProps as StandardProps, Theme } from '..';
+import { Theme } from '..';
 import { RatingClasses } from './ratingClasses';
+import { OverridableComponent, OverrideProps } from '../OverridableComponent';
 
 export interface IconContainerProps extends React.HTMLAttributes<HTMLSpanElement> {
   value: number;
 }
 
 export interface RatingPropsSizeOverrides {}
 
-export interface RatingProps
-  extends StandardProps<React.HTMLAttributes<HTMLSpanElement>, 'children' | 'onChange'> {
```

Comment:
This felt unncessary, from my understanding, it looks like `StandardProps` is removing `classes` prop and whatever types passed in 2nd arg. Even ater removing type of both `onChange` and `children` is uneffected.

Tests in this file seems to confirm it. https://github.com/mui/material-ui/pull/44199/files#diff-d2ea835e807d4687379ca84a765aad7416671765d6add33c16ef46bd6bd1499c